paper/paper.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Game Theory and Python: An educational tutorial to game theory and
+repeated games using Python"
+tags:
+- game theory
+- repeated games
+- prisoner's dilemma
+- axelrod python
+authors:
+- name: Nikoleta E. Glynatsi
+- name: Dr Vincent A. Knight
+  ORCID: 
+  affiliation: 1 affiliations:
+- name: Department of Mathematics, Cardiff University, Senghennydd Rd, Cardiff CF24 4AG
+  index: 1, 2
+date: some date in 2019
+bibliography: paper.bib
+---
+
+# Summary
+
+Game Theory and Python is an open source educational tutorial aimed to
+introduce readers to game theory and more specifically to repeated games. The
+tutorial uses the open source library called
+[Axelrod-Python](https://github.com/Axelrod-Python/Axelrod) and [Jupyter
+Notebooks](https://jupyter.org) making this an open source, reproducible and
+interactive tutorial.
+
+# Statement of Need
+
+Game theory is a field of applied mathematics and an exceptional tool for
+introducing students to the field. Students can start off by playing games, then
+conceptualizing strategies, and before long using equations to formulate the
+interactions between players.
+
+Repeated games are a set of games that are particularly popular in educational
+game theory, and a specific game that has gained much attention in both research
+(Axelrod1981) and teaching (reference) is the Prisoner's Dilemma. The advantage
+of studying the Prisoner's Dilemma is that it models situations in which
+self-interest clashes with collective interest, thus it provides a powerful
+framework for illustrating the usage of mathematics in real life decision
+making.
+
+% TODO a good reference that the PD is heavily used in education.
+
+# Content
+
+The four notebooks of this tutorial focus on the iterated version of the
```

Comment:
`These materials consist of four tutorials that focus on the iterated version of ...`